SPECS/gpumon.spec

```
@@ -53,6 +53,9 @@ DESTDIR=%{buildroot} %{__make} install
 %{_unitdir}/xcp-rrdd-gpumon.service
 
 %changelog
+* Fri Nov 03 2023 Guillaume Thouvenin <guillaume.thouvenin@vates.tech> - 0.18.0-11.2
+- Don't output to stdout/stderr (and thus daemon.log): they are already logged by RRDD
```

My change was not perfect: "they" referred to logs, but I removed the word "logs". New attempt.

```suggestion
- Don't output to stdout/stderr (and thus daemon.log): there already are RRDD logs
```

---

that's fine for me :+1: